default/stringtables/en.txt

```diff
@@ -4219,6 +4219,39 @@ Next are icons showing the player empire total [[encyclopedia INDUSTRY_TITLE]],
 
 On the right hand side of the control bar are a number of icons for toggling display of other significant UI windows. Tooltips are available for these.'''
 
+DESIGN_WINDOW_ARTICLE_TITLE
+Design Window
+DESIGN_WINDOW_ARTICLE_TEXT
+'''The design window is for viewing or altering the designs of monsters or ships.
+
+There are four sub windows:  Design Starts, Ship Parts, Design Details and the Pedia.
+
+
+Design Starts has four tabs: empty Hulls, Finished designs, Saved designs from previous games and Monster designs for inspection.  Drag or double click hulls, designs or monsters to edit/examine them on the workbench.
+
+Finished designs appear in the same order as on the Production Window.  Drag and drop designs within the list to change the order.  Delete designs with Obsolete from the context menu.  Ships already queued for production will continue to be built with the old design.  Save designs to disk for future games with Save from the context menu.
```

Comment:
oddly mixes imperative and descriptive sentences. should be in separate paragraphs, or precede the instructions with "you can", or better yet, rephrase to "Designs can be drag-and-dropped within the list to change their order."